client/app/views/MenuView.js

```diff
@@ -225,6 +225,23 @@ class MenuView extends Component {
         return false;
     }
 
+    // Shows the meal tabs if there are any available, otherwise shows nothing.
+    renderDynamicTabs() {
+        const tabs = this.dynamicTabButtons();
+        return (tabs.length == 0 
+            ? (
+                <View style={{...styles.topTabs.withPaddingBottom}}></View>
```

Comment:
Can you
`git checkout master`
`git pull`
`git checkout [this-branch]`
`git merge master`

and test again? `...styles.topTabs` was changed to `...styles.container` in the last commit for abstraction's sake.